sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/Storage.java

```diff
@@ -96,7 +99,7 @@ static synchronized void unsetInstance() {
      * @param apiUrl API base URL.
      */
     @SuppressWarnings({"SameParameterValue", "WeakerAccess", "unused"})
-    // TODO Remove suppress warnings after reflection removed in test app
+    // TODO (guperrot): Remove suppress warnings after reflection removed in test app.
```

Comment:
uh no, never use any alias in code please, also we work as a team, I'm not assigned tasks in comments.